build.gradle

```diff
@@ -37,6 +37,7 @@ allprojects {
     jcenter()
     maven { url 'https://plugins.gradle.org/m2' }
     maven { url 'https://mapbox.bintray.com/mapbox' }
+//    maven { url 'https://oss.jfrog.org/artifactory/oss-snapshot-local/' }
```

Comment:
Minor thing: Should we remove?

Reply:
I left it for ease of use in the future. So that we don't have to look up the reference every time :sweat_smile: